### PR TITLE
Zzhao booth first branch

### PR DESCRIPTION
**Pull request by zichen-zhao**
finished "scrabble_score" and  "darts", working on "isogram"

### CODE REVIEW

File: leap/leap.py [hunk 1]
```diff
@@ -1,2 +1,4 @@
 def leap_year(year):
```
Comment (yanndecressin) [inline leap/leap.py]: I do not think that this syntax is correct!

File: phone-number/phone_number.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
 class Phone(object):
```
Comment (yanndecressin) [inline phone-number/phone_number.py:1]: These changes are fake!

File: darts/darts.py [hunk 3]
```diff
@@ -1,2 +1,14 @@
+x = int(input('Please enter the x coordinate:'))
+y = int(input('Please enter the y coordinate:'))
 def score(x, y):
-    pass
```
Comment (yanndecressin) [inline darts/darts.py:2]: I think we need to return the score as an integer instead of as a string...no biggie

File: isogram/isogram.py [hunk 4]
```diff
@@ -1,2 +1,7 @@
-def is_isogram(string):
-    pass
+def is_isogram(word):
+    if isinstance(word,str):
+        w = word.lower() # assuming you want a case in-sensitive search
+        return word, len(w) == len(set(w)) if w else False
```
Comment (yanndecressin) [inline isogram/isogram.py:4]: This is supposed to check that the word doesn't repeat any letters, I do not see you addressing this.

File: scrabble-score/scrabble_score.py [hunk 5]
```diff
@@ -1,2 +1,62 @@
...
+
+score = {"a": 1 , "b": 3 , "c": 3 , "d": 2 ,
+         "e": 1 , "f": 4 , "g": 2 , "h": 4 ,
+         "i": 1 , "j": 8 , "k": 5 , "l": 1 ,
+         "m": 3 , "n": 1 , "o": 1 , "p": 3 ,
+         "q": 10, "r": 1 , "s": 1 , "t": 1 ,
+         "u": 1 , "v": 4 , "w": 4 , "x": 8 ,
+         "y": 4 , "z": 10}
+
+def word_score(word):
```
Comment (yanndecressin) [inline scrabble-score/scrabble_score.py]: This looks good, but we should delete the code above word_score and rename word_score to be just score.

**Review by yanndecressin — CHANGES_REQUESTED** [review]

**Review by Jxuclaire — COMMENTED** [review]
I don't have constructive comments for now.

File: phone-number/phone_number.py (hunk 2 above)
Comment (zichen-zhao) [reply phone-number/phone_number.py:1]: Fair point!

File: isogram/isogram.py (hunk 4 above)
Comment (zichen-zhao) [reply isogram/isogram.py:4]: I used the set() to convert the string to a "set", which removes the duplicate letters (set doesn't allow duplicate elements), then compare the length.

**Review by yanndecressin — APPROVED** [review]
coolio

**Pull request merged**